src/lib/AutocompleteTextInput.svelte

```diff
@@ -0,0 +1,21 @@
+<script lang="ts">
+    import { FormElement } from "./index.js";
+  
+    export let label: string;
+    export let name: string;

```

Comment:
The input API could be improved. We never want people to have to specify UUIDs, or else we'll have a repeat of that bug from months ago. Other components here generate with `uuidv4`.

By exposing the data list element, it also looks like the caller will be forced to do work to set things up. How about we had something like

```
export let label: string;
export let value: string;
export let autocomplete: string[];
```

and this component does the work of making the datalist? https://svelte.dev/repl/2230a1ec6a374c9da13a4e55a1e8701c?version=3.50.1